test/integration/usecase_vpn_test.go

```diff
@@ -125,8 +125,8 @@ func testVPN(t *testing.T, ptnum, nodesCount int, affinity map[string]int, verbo
 	}
 
 	pingCommand := "ping"
-	addressPool := "10.60.1.0/24"
-	srcIP, dstIP := "10.60.1.1", "10.60.1.2"
+	addressPool := "172.16.1.0/24"
+	srcIP, dstIP := "172.16.1.1", "172.16.1.2"
```

Comment:
@denis-tingajkin can you elaborate in the PR body why this is the fix?

Reply:
Also could we check this dynamically for test to choose exclude prefixes for cluster we are running on?

Reply:
Ok discussed, according to @lobkovilya we already have tests with same values, and in future we could think about moving to dynamic obtaining of this values.